app/products.py

```diff
@@ -0,0 +1,114 @@
+"""
+ Must have the environment variables set properly prior to running the script
+"""
+from woocommerce import API
+import json
+import os
+from faker import Faker
+import random
+
+fake = Faker()
+
+
+def create_api_object():
+    wcapi = API(
+        url=os.environ.get('BASE_URL'),
+        consumer_key=os.environ.get('API_KEY'),
+        consumer_secret=os.environ.get('API_SECRET'),
+        version="wc/v3",
+        timeout=15
+    )
+    return wcapi
+
+
+store_api = create_api_object()
+
+
+class Products:
+    def __init__(self) -> None:
+        self.endpoint = 'products'
+        self.products = self.fetch_all_products()
+
```

Comment:
What you did is fine.
But if it was me I would create the API helper object in here.
`self.wcapi = API(.....)`